addons/im_livechat/tests/test_get_operator.py

```diff
@@ -221,3 +223,70 @@ def test_priority_by_number_of_chat_all_operators_exceed_limit(self):
         self._create_chat(livechat_channel, second_operator)
         self._create_chat(livechat_channel, second_operator)
         self.assertEqual(first_operator, livechat_channel._get_operator())
+
+    def test_max_concurent_sessions(self):
+        operator = self._create_operator()
+        livechat_channel = self.env["im_livechat.channel"].create(
+            {
+                "name": "Livechat Channel",
+                "user_ids": [operator.id],
+                "concurent_session": True,
+                "concurent_session_number": 2,
+            }
+        )
+        self._create_chat(livechat_channel, operator)
+        self._create_chat(livechat_channel, operator)
+        self.assertEqual(False, livechat_channel._get_operator())
+
+    def test_max_concurent_sessions_expired(self):
+        operator = self._create_operator()
+        livechat_channel = self.env["im_livechat.channel"].create(
+            {
+                "name": "Livechat Channel",
+                "user_ids": [operator.id],
+                "concurent_session": True,
+                "concurent_session_number": 2,
+            }
+        )
+        self._create_chat(livechat_channel, operator)
+        self.env["discuss.channel"].create(
+            {
+                "name": "Visitor 1",
+                "channel_type": "livechat",
+                "livechat_active": True,
+                "livechat_channel_id": livechat_channel.id,
+                "livechat_operator_id": operator.partner_id.id,
+                "channel_member_ids": [Command.create({"partner_id": operator.partner_id.id})],
+                "last_interest_dt": fields.Datetime.now() - timedelta(minutes=31),
+            }
+        )
+        self.assertEqual(operator, livechat_channel._get_operator())
+
+    def test_max_concurent_sessions_multi_operators(self):
```

Comment:
I believe this test does not provide much value and could be coupled with the test I suggested previously ("available_operator_id").